modules/041-linstor/images/piraeus-operator/patches/README.md

```diff
@@ -1,5 +1,9 @@
 ## Patches
 
+### Add SELinux params for enforced mode
+
+This is proposition from upstream, made for set satellite usable in enforced SELinux mode. 
+
```

Comment:
Please add link to upstream PR

https://github.com/piraeusdatastore/piraeus-operator/pull/477